Build error missing "qr_encode.h"

<!--

Have you read Marlin's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.0.x/.github/code_of_conduct.md

Do you want to ask a question? Are you looking for support? Please don't post here. Instead use one of the following options:

- The Marlin Firmware forum at https://reprap.org/forum/list.php?415
- The MarlinFirmware Facebook Group at https://www.facebook.com/groups/1049718498464482/
- The MarlinFirmware Discord Server at https://discord.gg/n5NJ59y.

Before filing an issue be sure to test the latest "bugfix" branch to see whether the issue is already addressed.

-->

### Description

During the build for Robin Nano 1.x following he instructions, the following error appear:
Marlin/src/lcd/extui/lib/mks_ui/draw_cloud_bind.cpp:36:10: fatal error: qr_encode.h: No such file or directory

### Steps to Reproduce

1. Install VS Code
2. Install Platform IO IDE extension from VS
3. Clone Mks-Robin-Nano-Marlin2.0-Firmware
4. Edit Configuation.h as follows:

       #define MOTHERBOARD BOARD_MKS_ROBIN_NANO

       #define TFT_LVGL_UI_FSMC

       //#define TFT_LVGL_UI_SPI

5. using Platform IO to build for env:mks_robin_nano35

**Expected behavior:** [What you expect to happen]
 successful build process

**Actual behavior:** [What actually happens]
 compilation terminated with the following errors:
      Marlin/src/lcd/extui/lib/mks_ui/draw_cloud_bind.cpp:36:10: fatal error: qr_encode.h: No such file or directory
      ........
     *** [.pio/build/mks_robin_nano35/src/src/lcd/extui/lib/mks_ui/draw_cloud_bind.cpp.o] Error 1
![Screenshot from 2020-11-08 00-02-11](https://user-images.githubusercontent.com/8094993/98451348-cc646380-2155-11eb-9649-31be051e9707.png)
[Marlin.zip](https://github.com/makerbase-mks/Mks-Robin-Nano-Marlin2.0-Firmware/files/5505422/Marlin.zip)


#### Additional Information

* Include a ZIP file containing your `Configuration.h` and `Configuration_adv.h` files.
* Provide pictures or links to videos that clearly demonstrate the issue.
* See [How Can I Contribute](#how-can-i-contribute) for additional guidelines.
[Marlin.zip](https://github.com/makerbase-mks/Mks-Robin-Nano-Marlin2.0-Firmware/files/5505423/Marlin.zip)
![Screenshot from 2020-11-08 00-02-11](https://user-images.githubusercontent.com/8094993/98451364-ee5de600-2155-11eb-91b3-17b3e587bd1d.png)

--- a/Marlin/src/lcd/extui/lib/mks_ui/draw_cloud_bind.cpp
+++ b/Marlin/src/lcd/extui/lib/mks_ui/draw_cloud_bind.cpp
@@ -33,7 +33,7 @@
 #include "../../../../MarlinCore.h"
 #include "../../../../module/temperature.h"
 
-#include "qr_encode.h"
+#include "QR_Encode.h"
 
 extern lv_group_t * g;
 static lv_obj_t * scr;

